### DISCUSSION TITLE
Shootings by Cops in USA

### DISCUSSION THREAD

**Discussion by VkSath00** [jakobzhao/geog495#57] [Final Project]
Repo: https://github.com/VkSath00/Final_Project_Vksatham
Link: https://vksath00.github.io/Final_Project_Vksatham/

Description - 
This project is responsible for displaying the total shootings by Police Officers throughout the country since 01-01-2015. This heatmap displays the trends where more cop related shootings occur. Due to the recent investigations of police officers shooting unarmed suspects, this map allows for the user to select any shooting incident and know if the suspect was armed and with what weapon they were armed with. (Not done yet)
![heatmap](https://user-images.githubusercontent.com/49254474/145764459-7feef0d9-9219-4665-88e6-1769b34bbc54.JPG)
![armed](https://user-images.githubusercontent.com/49254474/145764717-c1ca19bb-8d67-4a29-9ad5-bd34dfd1ff32.JPG)

**Comment by Conibaer** [comment]
Your map is very interesting, proven by the fact I spent a fair amount of time zooming around different cities in the US. One thing to include would be some information about the input data used like what time span it covers and what year the data is from. This background information about the data can be very useful for understanding patterns in greater depth.

**Comment by notzulkifli** [comment]
The back mapbox and heat map color work really well together. I like how the heatmap color really stands out.  This map does a really great job in describing a different view of what us in the western side of the united states often don't see which is shooting by cops. If i may suggest one thing, i would suggest having a legend and to show what the different size heat maps plots represent, and when you click on the heatmap points it shows when the shooting occured. I like the idea that you are going for with the "armed" or "unarmed" in the popup!
